lib/amazon-pricing/definitions/database-type.rb

```diff
@@ -13,39 +13,126 @@
 module AwsPricing
   class DatabaseType < CategoryType
 
+    # Display Name, as seen in DBR/CUR
+    MYSQL_STANDARD            = 'MySQL Community Edition'.freeze
```

Comment:
can we double check - feels like we missed one DB

Reply:
I just counted # of lines previously and after. If you can verify the count of DBs is the same (with the addition of Aurora postgres) then this is not an issue